tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenCreateIndexTest.cs

```diff
@@ -24,16 +24,19 @@ namespace MongoDB.Driver.Tests.JsonDrivenTests
     public sealed class JsonDrivenCreateIndexTest : JsonDrivenCollectionTest
     {
         // private fields
-        private CreateIndexModel<BsonDocument> _createIndexModel;
+        private IndexKeysDefinition<BsonDocument> _indexKeys;
+        private string _indexName;
         private IClientSessionHandle _session;
 
-        // public constructors
-        public JsonDrivenCreateIndexTest(IMongoCollection<BsonDocument> collection, Dictionary<string, object> objectMap)
-            : base(collection, objectMap)
+        // properties
+        private CreateIndexModel<BsonDocument> CreateIndexModel
         {
+            get { return new CreateIndexModel<BsonDocument>(_indexKeys, new CreateIndexOptions { Name = _indexName }); }
```

Comment:
![image](https://user-images.githubusercontent.com/55145764/78380871-383d8700-75dd-11ea-9ec3-2ee43acc464c.png)

Reply:
I wouldn't worry about Lazy or not. Just create it every time. But move it to a private method (at the end of the file):

    // private methods
    private CreateIndexModel<BsonDocument> CreateCreateIndexModel()
    {
        var keysDefinition = new BsonDocumentIndexKeysDefinition<BsonDocument>(_keys);
        var options = new CreateIndexOptions { Name = _name };
        return new CreateIndexModel<BsonDocument>(keysDefinition, options);
    }

Reply:
Done.